src/viewer/viewer.ts

```diff
@@ -483,7 +483,8 @@ export default class Viewer {
     // workaround to reset the gl state after using testTextureSupport
     // fixes some bug where nothing is rendered to the canvas
     // when animations are started on page load
-    this.renderer.clearTarget(this.pickingTarget, true, true, true)
```

Comment:
Could you explain why you made this change? Apparently clearTarget() is still a valid method of renderer, I am wondering if there is an advantage?

Reply:
clearTarget is deprecated (as of r97), so I thought it best to remove it. From the [migration guide](https://github.com/mrdoob/three.js/wiki/Migration-Guide#r96--r97):

> WebGLRenderer.clearTarget() is now deprecated. Use WebGLRenderer.setRenderTarget() in combination with WebGLRenderer.clear() instead.

It produces a warning on the console if you use it.

Reply:
It's deprecated and prints a warning. See the [migration guide](https://github.com/mrdoob/three.js/wiki/Migration-Guide#r96--r97).

Reply:
Ok, cool